src/frontend/Router/router.js

```diff
@@ -0,0 +1,21 @@
+import { Routes, Route } from "react-router-dom";
+import { Home, Login, Signup } from "../pages/index";
+import Mockman from "mockman-js";
+import { RequiresAuth } from "./RequiresAuth";
+
+export function Router() {
+  return (
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/login" element={<Login />} />
+      <Route path="/signup" element={<Signup />} />
+      <Route path="/test-api" element={<Mockman />} />
+      {/* <Route path="*" element={<PageNotFound />} />
+      <Route path="/bookmarks" element={<Bookmarks />} />
+      <Route path="/profile" element={<Profile />} />
+      <Route path="/userFeed" element={<UserFeed />} />
+      <Route path="/explore" element={<Explore />} />
+      <Route path="/post" element={<SinglePost />} /> */}
+    </Routes>
```

Comment:
you can remove commented code if not needed

Reply:
Will remove all the consoles in the very last commit.